code/app/src/main/res/navigation/mobile_navigation.xml

```diff
@@ -147,4 +150,14 @@
             android:id="@+id/action_eventBrowseFragment_to_eventDetailsFragment"
             app:destination="@id/eventDetailsFragment" />
     </fragment>
+    <fragment
+        android:id="@+id/mapsFragment"
+        android:name="com.example.the_tarlords.ui.MapsFragment"
+        android:label="MapsFragment">
+        <action
+            android:id="@+id/action_mapsFragment_to_eventDetailsFragment"
```

Comment:
for going backwards, I believe it worked better for going back and forth between fragments if the return action was an action_fragment_pop (done by right click on fragment in design view -> add action -> return to source)